.github/workflows/tox-pytest.yml

```diff
@@ -0,0 +1,73 @@
+name: tox-pytest
+
+on: [ push, pull_request ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Compile Zenodo Datastore DOIs
+      run: grep -e '.*10\.\(5281\|5072\)/zenodo\..*' src/pudl/workspace/datastore.py | sed -e 's/",*$//g' | sed -e 's/^.*"//g' | sort > datastore-dois.txt
```

Comment:
This is a temporary hack, to grab the list of all the Zenodo DOIs and dump them in a file, so we can tell if the input data has changed (with the hashFiles() call later) to allow caching of the inputs.  I'd like to move the DOIs into a JSON file in the `package_data` to avoid this.